crates/bevy_ecs/src/reflect/entity_commands.rs

```diff
@@ -13,16 +13,17 @@ use std::marker::PhantomData;
 
 /// An extension trait for [`EntityCommands`] for reflection related functions
 pub trait ReflectCommandExt {
-    /// Adds the given boxed reflect component to the entity using the reflection data in
+    /// Adds the given boxed reflect component or bundle to the entity using the reflection data in
     /// [`AppTypeRegistry`].
     ///
-    /// This will overwrite any previous component of the same type.
+    /// This will overwrite any previous component(s) of the same type.
     ///
     /// # Panics
     ///
     /// - If the entity doesn't exist.
-    /// - If [`AppTypeRegistry`] does not have the reflection data for the given [`Component`](crate::component::Component).
-    /// - If the component data is invalid. See [`PartialReflect::apply`] for further details.
+    /// - If [`AppTypeRegistry`] does not have the reflection data for the given
+    /// [`Component`](crate::component::Component) or [`Bundle`](crate::bundle::Bundle).
+    /// - If the component or bundle data is invalid. See [`PartialReflect::apply`] for further details.
```

Comment:
In my opinion, it would be good to also write how to handle errors.

Reply:
Since this is a panic, would you not handle it like you would handle other bevy panics? Not familiar with how to handle panics myself, but if there's some documentation for it we can link it here

Reply:
Yeah, we don't have specific guidance here currently; I'm not sure there's more to add.